docs/nuvla/user-guide/run-app.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: page
+title: Launch Applications
+parent: User Guide
+grand_parent: Nuvla
+nav_order: 9
+permalink: /nuvla/launch-app
+---
+
+# Running a simple applications
+
+To run a simple application on a NuvlaBox, [go here](/nuvlabox/deploying-apps/first-app).
```

Comment:
@mebster is this section going to be expanded? Cause if not, it seems a bit weird that Nuvla's key feature is forwarding me to a NuvlaBox section

Reply:
This is a stop gap. We need a section in Nuvla to launch an app. And since this is the closest we currently have, I linked to it.  I think it would make sense to have a vanilla launch app section, which the NuvlaBox section could include. Then we would have it on both sides without markdown duplication.

Reply:
yep makes sense. Please create a ticket for that and we'll merge this as it is for now